docs/how-to/solve-with-regret-minimization.rst

```diff
@@ -0,0 +1,34 @@
+.. _how-to-use-regret-minimization:
+
+Solve with support enumeration
```

Comment:
```suggestion
Solve with Regret Minimization
```

Reply:
Gosh, I don't know how I missed it; copy-paste an error—changed in the latest commit.